src/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -27,6 +27,8 @@ public class HttpRequestMessage : IDisposable
         private bool _disposed;
         private IDictionary<string, object> _properties;
 
+        internal bool SendConcurent;
```

Comment:
Why do we need this?

Reply:
+1. The handler itself shouldn't change behavior based on whether ResponseHeadersRead is specified or not.

Reply:
I was wondering about that. It was not clear if not waiting for sending request can introduce any compatibility issue and it is what we do HTTP/1.1.
It also has best way how to propagate errors to caller IMHO. I can certainly remove this.